test/jdk/java/awt/Frame/DisposeTest.java

```diff
@@ -39,11 +39,13 @@
  * @bug 4127271
  * @summary Tests that disposing of a Frame with MenuBar removes all traces
  *          of the Frame from the screen.
+ * @run main/othervm -Dsun.java2d.uiScale=1 DisposeTest
```

Comment:
Why do you need to set scale factor=1, does the default break something?

Reply:
I had added uiScale=1 as a defensive fix but since it doesn't fail on CI without it, I have gone ahead and removed it.